src/ripple/app/paths/impl/BookStep.cpp

```diff
@@ -561,6 +569,14 @@ BookStep<TIn, TOut>::fwdImp (
 
         remainingIn = in - result.in;
         this->consumeOffer (sb, offer, ofrAdjAmt, stpAdjAmt, ownerGivesAdj);
+
+        // When the mantissas of two iou amounts differ by less than ten, then
+        // subtracting them leaves a result of zero. This can cause the check for
+        // (stpAmt.in > remainingIn) to incorrectly think an offer will be funded
+        // after subtracting remainingIn.
+        if (amendmentRIPD1298(sb.parentCloseTime()))
+            processMore = processMore || offer.fully_consumed();
```

Comment:
`|=` is not short-circuited when `processMore` is already true. We'll save a function call.